tests/console/validate_nfs_repo.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that, after remote installation, the VNC connection
+# and the NFS repositories are available.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+
+use testapi;
+use lockapi;
+
+sub run {
+    mutex_create("remote_first_boot");
+    select_console "root-console";
+    # Check if there are NFS repositories
+    assert_script_run "zypper lr -u | grep -i nfs";
```

Comment:
recently we are using this library function [validate_repo_properties](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/fbe424869d93afd1692f9ddde00d7d0005c22542/tests/console/validate_dud_addon_repos.pm#L29) quite a lot for validation of repos.

Reply:
Done, waiting for vrs.